docs/source/installation/index.md

```diff
@@ -186,13 +187,18 @@ default_calc_job_plugin='quantumespresso.pw',
 
 :::{important}
 Make sure to replace `/path/to/pw.x` with the actual absolute path to the `pw.x` binary.
+If the binary is in your `PATH`, you can find the first `pw.x` binary in your `PATH` directories using the `which` command:
+
+```console
+which pw.x
+```
```

Comment:
The path actually no longer has to be absolute (since v2.3) so I think we should just recommend to specify `pw.x`. The current version of `aiida-quantumespresso` requires `aiida-core~=2.3`.

Reply:
Good to know, but maybe it's still good policy to set the _full_ path? A user might have multiple `pw.x` binaries lying around, and this way they put some thought into using the one they actually want to run.

Reply:
Maybe we can just leave the relative path as the first suggestion and then in a note or warning box below it, give more info about how to pass an absolute path if you want to be sure which executable to call.